### ISSUE TITLE
Abort Trap Error on Assigning Anonymous Classes to Properties

### ISSUE THREAD

**Issue by tpunt** [krakjoe/pthreads#659]
Environment
==========

  * PHP: PHP 7.1.0 (cli) ( ZTS DEBUG )
  * pthreads: 3.1.7dev
  * OS: Mac

Reproducing Code
==============

```php
$task = new class extends Thread {
    public function run()
    {
        $this->prop = new class {};
    }
};

$task->start() && $task->join();
```

**Comment by krakjoe** [comment]
I had a go at #658 and #659 this morning ... but failed ... I'll have to come back to it ...

**Comment by tpunt** [comment]
Resolved by #686.
